content/docs/1.1.1/deploy/install/_index.md

```diff
@@ -6,9 +6,9 @@ weight: 1
 
 Longhorn can be installed on a Kubernetes cluster in several ways:
 
+- [Rancher catalog app](./install-with-rancher)
 - [kubectl](./install-with-kubectl/)
 - [Helm](./install-with-helm/)
-- [Rancher catalog app](./install-with-rancher)
```

Comment:
This aligns with the website left layout.

![image](https://user-images.githubusercontent.com/49380831/111744544-5e2f6a80-88c6-11eb-9649-871f80f3d0d9.png)